module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -338,26 +337,32 @@ namespace module::platform::darwin {
                     }
                 }
 
+                // It's worth making absolutely sure that these are correct for a NUgus...
                 // gyro_x to the right
```

Comment:
IMO This block of comments is correct, The Gyro is right handed and the accelerometer is left handed